### DISCUSSION TITLE
my github username is displayed differently on the github site

### DISCUSSION THREAD

**Discussion by gigadev0915** [community/community#112450] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I use github desktop when I push to github.
But recently I saw that my github username is displayed differently on the github site.
My username is codebrain0318, but they show that yuriiyeskov112 pushed.
Please help me solve this issue.
Thank you.

**Comment by AidenJamesYt** [comment]
make sure that your local Git configurations are set correctly with the desired username and email address. Heres how to do so:

Open your terminal or command prompt and run the following commands, replacing "Your Name" and "your.email@example.com" with your actual name and email address:

`git config --global user.name "Your Name"
git config --global user.email "your.email@example.com"`

This ensures that your commits are attributed to the correct GitHub account.

Let me know if it works or not @codebrain0318

**Reply by gigadev0915** [reply]
thank you for your answer,
but everything is normal.
![Screenshot_1](https://github.com/community/community/assets/136085062/a065956d-6275-4ecd-93da-4b4662201672)

**Reply by gigadev0915** [reply]
![Screenshot_2](https://github.com/community/community/assets/136085062/d6c5c9c6-c7ab-4c17-bc13-56dcaad675a6)
![Screenshot_3](https://github.com/community/community/assets/136085062/02204a61-91f9-4a6c-b085-98583982b483)

**Reply by AidenJamesYt** [reply]
did you make the edits on that account?
